src/confd/yang/infix-if-vlan@2023-10-25.yang

```diff
@@ -0,0 +1,65 @@
+submodule infix-if-vlan {
+  yang-version 1.1;
+  belongs-to infix-interfaces {
+    prefix infix-if;
+  }
+
+  import ietf-interfaces {
+    prefix if;
+    reference
+      "RFC 8343: A YANG Data Model For Interface Management";
+  }
+
+  import infix-if-type {
+    prefix infixift;
+  }
+  import ieee802-dot1q-types {
+    prefix dot1q-types;
+    revision-date 2022-10-29;
+    reference
+      "IEEE Std 802.1Q-2022: IEEE Standard for Local and
+       metropolitan area networks -- Bridges and Bridged Networks";
+  }
+  contact      "kernelkit@googlegroups.com";
+  description
+    "This module implements VLAN (8021q) encapsulation";
+
+  revision 2023-10-25 {
+    description "Initial revision";
+  }
+
+  augment "/if:interfaces/if:interface" {
+    when "derived-from-or-self(if:type, 'infixift:vlan')" {
+      description "Only shown for if:type vlan";
+    }
+    description "Augment to add  802.1Q VLAN tag classifications";
+    container vlan {
+      description
+        "Classifies 802.1Q VLAN tagged Ethernet frames to a
+           interface  by exactly matching the
+           tag type(s) and VLAN identifier(s).
+
+           Only frames matching the classification configured on a
+           interface are processed on that
+           interface.";
```

Comment:
Please line up multi-line descriptions under the first column of the first line, and reflow each paragraph to a "sane amount of charachters".   For an example, see the description text for `ieee-reserved-groups` in `infix-if-bridge.yang`. The CLI help text for `help vlan` uses the original model formatting.